frontend/src/features/documents/documentDetail/DocumentDetailView.tsx

```diff
@@ -53,29 +78,125 @@ const DocumentDetailView: React.FunctionComponent<IDocumentDetailsViewProps> = p
           <StyledGreySection>
             <Row className="pb-3">
               <Col className="text-left">
-                <StyledH2>Document information</StyledH2>
+                <StyledHeader>
+                  <StyledH2>Document information</StyledH2>
+                  <TooltipIcon
+                    toolTipId={'documentInfoToolTip'}
+                    className={'documentInfoToolTip'}
+                    toolTip="Information you provided here will be searchable"
+                  ></TooltipIcon>
+                </StyledHeader>
               </Col>
-              {hasClaim(Claims.DOCUMENT_EDIT) && <Col xs="2">Edit</Col>}
-            </Row>
-            <SectionField label="Status" labelWidth="4">
-              {props.document.pimsDocument?.statusTypeCode?.description}
-            </SectionField>
-
-            <StyledH3>Details</StyledH3>
-            <StyledScrollable>
-              {props.document.mayanMetadata?.length === 0 && (
-                <StyledNoData>No additional data</StyledNoData>
+              {hasClaim(Claims.DOCUMENT_EDIT) && (
```

Comment:
If the screen is already in edit mode should this icon be displayed?

Reply:
now hidden if editable mode